### PR TITLE
Addition of Outline for 3/27/23

### PR DESCRIPTION
**Pull request by jaxlub**

### CODE REVIEW

**Comment by kevinaangstadt** [comment]
@glendalyys please review this PR in the comments

File: Day18Notes.md
Comment (glendalyys) [inline Day18Notes.md:1]: Hi Jax here is my review of your outline:
- I don't think you need to include your name in the header. Per the style guide I think we just have to have the header say Day X Notes.
- Don't forget to include the note overview
- Stylistically, you should try to separate each section with thicker lines as per the style guide. 
- Also Definitions should be in block quotes with the word bolded. 
- I think you can make the bug report a list so it's a bit clearer 
- I'd separate this into sections the first being the one you have and the second about a "bug life cycle". This way you can break up the question like how it is in the notes. 

All the best,
Glen

**Comment by kevinaangstadt** [comment]
@jaxlub please make the requested changes

**Comment by jaxlub** [comment]
@glendalyys @kevinaangstadt Thank you for the input! I have updated the file based off what you requested to meet the style guide specifications. Let me know if there is anything else you would like me to correct. :)

**Pull request merged**